### PR TITLE
add-book-entity-service-repository

### PR DESCRIPTION
**Pull request by andrey-kiyas**
Added:
- book entity
- book service
- book repository

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,24 +30,47 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+            <version>3.1.2</version>
```
Comment (Rommelua) [inline pom.xml]: use version property

File: pom.xml [hunk 2]
```diff
@@ -28,24 +30,47 @@
...
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.28</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-orm</artifactId>
+            <version>5.2.25.RELEASE</version>
+        </dependency>
```
Comment (Rommelua) [inline pom.xml]: why do you need it?

File: pom.xml [hunk 3]
```diff
@@ -28,24 +30,47 @@
...
     </dependencies>
 
     <build>
         <finalName>online-book-store</finalName>
         <plugins>
             <plugin>
-                <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-maven-plugin</artifactId>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
```
Comment (Rommelua) [inline pom.xml]: you do not need war plugin

File: src/main/java/com/onlinebookstore/model/Book.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    @Column(nullable = false)
+    private BigDecimal price;
+    private BigDecimal description;
+    private BigDecimal coverImage;
```
Comment (Rommelua) [inline src/main/java/com/onlinebookstore/model/Book.java]: BigDecimal ?

**Review by Rommelua — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (andrey-kiyas) [reply pom.xml]: Without this plugin does not skip checkstyle

File: src/main/java/com/onlinebookstore/model/Book.java (hunk 4 above)
Comment (andrey-kiyas) [reply src/main/java/com/onlinebookstore/model/Book.java]: Sorry. Copy-paste. (

File: pom.xml (hunk 2 above)
Comment (andrey-kiyas) [reply pom.xml]: Done

File: pom.xml (hunk 1 above)
Comment (andrey-kiyas) [reply pom.xml]: Done

**Review by aaananas — APPROVED** [review]

**Pull request merged**